toolchains/scala/README.md

```diff
@@ -1,44 +1,133 @@
-# Scala toolchain to work with Chisel
+# Chisel Bazel Rules
 
-This toolchain is adapted to work with Chisel and support vscode with bloop.
+This directory contains Bazel rule wrappers for Chisel hardware design, built on top of [BCR rules_scala](https://github.com/bazelbuild/rules_scala).
 
-One of the problem with bazelbsp is that it invokes bazel, which locks up the GUI as well as the command line.
+## Architecture
 
-bloop on the other hand runs completely independently of Bazel inside vscode
+The project uses **BCR rules_scala 7.1.5** from Bazel Central Registry as the underlying Scala toolchain. The Chisel wrappers (`chisel_binary`, `chisel_library`, `chisel_test`) provide a convenient API that:
```

Comment:
exact version is in MODULE.bazel, I'd write "7.1.5 or newer as per MODULE.bazel"